test/i18next_test.dart

```diff
@@ -781,4 +781,122 @@ void main() {
       expect(I18Next.of(capturedContext!), isNotNull);
     });
   });
+
+  group('orElse', () {
+    group('fallback value', () {
+      const fallback = 'Fallback';
+      test('when key is not found', () {
+        expect(
+          i18next.t('$namespace:key', orElse: (key) => fallback),
```

Comment:
Could you also wrap the `orElse` callback with:

```dart
expectAsync1((key) {
  expect(key, '$namespace:key');
  return fallback;
})`
```

just to check that it is called exactly once and that the key arg is correct?
(adding it just to this test case should suffice since all the others don't change the behavior)

Reply:
Right, thank you for the pointer!

Reply:
Hey sorry for the delay, I was on vacation and will be back tomorrow to finalize the PR and release a new version!

Reply:
No problem, I figured this was the case :) 
Thank you for picking this up!